src/test/java/software/amazon/documentdb/jdbc/DocumentDbResultSetTest.java

```diff
@@ -349,7 +355,7 @@ void testVerifyRowVerifyColumnIndex() throws SQLException {
 
     @Test
     @DisplayName("Tests get from string")
-    void testGetString() throws SQLException {
+    void testGetString() throws SQLException, IOException {
         final String collection = "resultSetTestString";
         final Document document = Document.parse("{\"_id\": \"key1\"}");
         document.append("field", new BsonString("30"));
```

Comment:
Should we change/add a multibyte text string - to ensure we have Unicode support?